packages/wonder-blocks-search-field/src/components/search-field.js

```diff
@@ -0,0 +1,257 @@
+// @flow
+// A TextField with a search icon on its left side and X icon on its right side
```

Comment:
Could you please move this just before you declare the component (line # 104)? That way, Storybook Docs can get this description and add it to the Docs section:

https://5e1bf4b385e3fb0020b7073c-hxnnfisaue.chromatic.com/?path=/docs/searchfield--default

Here's a Cell example for reference: https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-cell/src/components/compact-cell.js#L10-L27
Output: https://khan.github.io/wonder-blocks/?path=/docs/cell-compactcell--default-compact-cell